tests/unit/test_utils.py

```diff
@@ -469,6 +469,13 @@ def test_auto_decode_utf16_le(self):
         )
         assert auto_decode(data) == "Django==1.4.2"
 
+    def test_auto_decode_utf16_be(self):
```

Comment:
Might be better to name this test `test_auto_decode_utf_16_be`. The current name is the same invalid abbreviation that you're fixing here :-)